src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -265,3 +268,39 @@ When migrating from our traditional Azure integrations to the Azure Monitor inte
 
 * When you enable an Azure Monitor integration, new separate entities will be created for all of your resources. The entities created by the Azure Polling integration are staying as they are. This means you must update dashboards, alerts, and any other capability that refrences those entities.
 * Old entities are available for 24 hours.
+
+
+## Migration steps from previous Azure Polling integrations[#migration-from-polling]
+
+On a typical deployment, migrating from Azure API Polling to the Azure Monitor involves the following steps (we recommend trying this on a dev / staging environment first):
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Azure and click "Manage Services"
+2. Enable the new "Azure Monitor metrics" integration and click the "Save Changes" button
+<img
+  title="Shows where to enable the azure monitor"
+  alt="Shows where to enable the azure monitor"
+  src={infrastructureAzureMonitorMetricsIntegration}
+/>
+
+3. The "Azure Monitor metrics" integration is now available
+<img
+  title="Shows where azure monitor metrics integration"
+  alt="Shows where azure monitor metrics integration"
+  src={infrastructureAzureMonitorMetricsConfigure}
+/>
+
+4. Enable and configure the new integration as described in [this section](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor/#set-up)
```

Comment:
This is a link to a different section of the same page, there must be a better way to do it